packages/zoe/src/contracts/priceAggregatorTypes.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @typedef {Object} OracleAdmin
+ * @property {() => Promise<void>} delete Remove the oracle from the aggregator
+ * @property {(result: any) => Promise<void>} pushResult rather than waiting for
+ * the polling query, push a result directly from this oracle
+ */
+
+/**
+ * @typedef {Object} PriceAggregatorCreatorFacet
+ * @property {(quoteMint: Mint) => Promise<void>} initializeQuoteMint
+ * @property {(oracleInstance: Instance, query: any=) => Promise<OracleAdmin>} initOracle
+ */
+
+/**
+ * @typedef {Object} PriceAggregatorPublicFacet
+ * @property {() => PriceAuthority} getPriceAuthority
+ */
+
+/**
+ * @typedef {Object} PriceAggregatorKit
+ * @property {PriceAggregatorPublicFacet} publicFacet
+ * @property {PriceAggregatorCreatorFacet} creatorFacet
+ */
+
+/**
+ * @typedef {Object} OraclePublicFacet the public methods accessible from the
+ * contract instance
+ * @property {(query: any) => ERef<Invitation>} makeQueryInvitation create an
+ * invitation for an oracle query
+ * @property {(query: any) => ERef<any>} query make an unpaid query
+ */
+
+/**
+ * @typedef {Object} OracleCreatorFacet the private methods accessible from the
+ * contract instance
+ * @property {() => AmountKeywordRecord} getCurrentFees get the current
+ * fee amounts
+ * @property {(total: boolean=) => ERef<Invitation>}
+ * makeWithdrawInvitation create an invitation to withdraw fees
```

Comment:
I prefer the declared name on the right end of the line after the curly. It's hard enough to visually scan these with their ragged placement, but after a curly stands out a little in the IDE.  If we consistently had the declared type left flush, that would be a great consistency, but when they're mostly on the right, I find these exceptions to be hard to find.

Reply:
This was also not a change made in this PR